server/src/main/java/org/opensearch/node/resource/tracker/ResourceTrackerSettings.java

```diff
@@ -26,6 +26,14 @@ private static class Defaults {
          * This is the default window duration on which the average resource utilization values will be calculated
          */
         private static final long WINDOW_DURATION_IN_SECONDS = 30;
+        /**
+         * This is the default polling interval for IO usage tracker
+         */
+        private static final long IO_POLLING_INTERVAL_IN_MILLIS = 5000;
+        /**
+         * This is the default window duration for IO usage tracker on which the average resource utilization values will be calculated
+         */
+        private static final long IO_WINDOW_DURATION_IN_SECONDS = 120;
```

Comment:
nit: Lets have a common unit

Reply:
Agree. But polling interval is 500 ms for CPU and Memory Trackers. So keeping it was ms and window duration is in sec. The window duration is dynamic setting here and it is always in sec.